dev-python/tikzplotlib/tikzplotlib-0.9.4.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+
+inherit distutils-r1 virtualx
+
+DESCRIPTION="Convert matplotlib figures into TikZ/PGFPlots"
+HOMEPAGE="https://github.com/nschloe/tikzplotlib"
+SRC_URI="https://github.com/nschloe/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	app-text/texlive
+	$( python_gen_cond_dep \
+		'dev-python/importlib_metadata[${PYTHON_USEDEP}]' python3_7 )
```

Comment:
So the importlib_metadata is only needed for python3.7 and not for other python versions?

Reply:
Yes, see upstream's setup.cfg file: https://github.com/nschloe/tikzplotlib/blob/master/setup.cfg

```
install_requires =
    importlib_metadata;python_version<"3.8"
    matplotlib >= 1.4.0
    numpy
    Pillow
```